devito/finite_differences/differentiable.py

```diff
@@ -133,6 +133,13 @@ def _subs(self, old, new, **hints):
                 continue
         return self.func(*args, evaluate=False)
 
+    def shift(self, dim, shift):
```

Comment:
can you move it below `def diff` ? here is right in between all the private methods...

Reply:
Thought it'd be good just next to `_subs` since it's basically a shortcut to it but sure.